games/chapter2/practice/moving_text.py

```diff
@@ -0,0 +1,23 @@
+# move some text!
+
+# The text should start in the upper corner and be moving
+# down and to the right. You can move the text using coordinates
+# or blitz the text on to a moving rectangle. Feel free to be
+# creative with colors, fonts, and font sizes. However, if
+# applicable, make the rectangle proportional to the text, and
+# everything smaller than the screen
+
+
+# Note: you can import a time module in your loop
+# to make it more clear
+
+
+# imports!
+
+SCREEN_SIZE = (800, 800)
+BLACK = (0, 0, 0)
+BLUE = (0, 0, 255)
+WHITE = (255, 255, 255)
+momentum = [2, 2]  # down and right
```

Comment:
Small nit: the `momentum` variable should probably be a tuple for consistency as you used a tuple in the other file.